mp-backend/src/handlers/recipe.ts

```diff
@@ -47,6 +47,15 @@ function isRecipeRequestBody(data: Record<string, unknown>): data is RecipeReque
     return !determineRecipeRequestBodyFields(data);
 }
 
+function getPrincipleId(event: APIGatewayProxyEvent): string {
```

Comment:
Please move this to middleware authenitcateJWT, or else...

Reply:
Done